lobby/src/main/java/org/triplea/lobby/server/login/LobbyLoginValidator.java

```diff
@@ -186,59 +186,26 @@ private static String getBanDurationBreakdown(final Timestamp stamp) {
   private @Nullable String authenticateRegisteredUser(
       final Map<String, String> response, final User user) {
     final String username = user.getUsername();
-    final String errorMessage = ErrorMessages.AUTHENTICATION_FAILED;
     final HashedPassword hashedPassword = database.getUserDao().getPassword(username);
     if (hashedPassword == null) {
-      return errorMessage;
+      return ErrorMessages.AUTHENTICATION_FAILED;
     }
+
     if (RsaAuthenticator.canProcessResponse(response)) {
       return rsaAuthenticator.decryptPasswordForAction(
           response,
           pass -> {
-            final String legacyHashedPassword =
-                response.get(LobbyLoginResponseKeys.HASHED_PASSWORD);
             if (hashedPassword.isBcrypted()) {
-              if (database.getUserDao().login(username, new HashedPassword(pass))) {
```

Comment:
The outer if statement is always executed, the inner if statement is always false as `legacyHashedPassword` is always null. The if statement on 201 is preserved.